### PR TITLE
 feat: added autosync for development and the new development-ci

### LINKED ISSUE #603: Autosync development-ci branch
Problem: right now when a bounty hunter opens a PR which updates the CI (example: https://github.com/ubiquity/ubiquity-dollar/pull/574) then we should:
1. Merge PR to the `development` branch
2. Check that the feature actually works

Ideally we should test such PRs without merging to the `development` branch.

So the expected flow for bounty hunters will be:
1. If the PR is connected with the CI then bounty hunter should open it to the `development-ci` branch first
2. We test the issue
3. The PR is reopened to the `development` branch and merged

What should be done:
1. Create a `development-ci` branch
2. Add a github action that autosyncs new updates from the `development` branch to the `development-ci` branch (example: https://github.com/pavlovcik/dao.ubq.fi/blob/677a373a263ce1ffd0239201f74bebfdac0e1f01/.github/workflows/sync.yml)
3. Update docs (or bounty hunter's readme) so that they know the desired flow for PRs connected with the CI

### PR DESCRIPTION
**Pull request by BeanieMen**
Resolves #603

This pull request is made to resolve the issue of commiting to the development branch directly

### CODE REVIEW

File: .github/workflows/development-ci.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Development to Development-CI Sync
+
+on:
```
Comment (rndquu) [inline .github/workflows/development-ci.yml:3]: Could you also add `workflow_dispatch` here?

**Review by rndquu — CHANGES_REQUESTED** [review]

File: .github/workflows/development-ci.yml (hunk 1 above)
Comment (BeanieMen) [reply .github/workflows/development-ci.yml:3]: sure

**Review by rndquu — APPROVED** [review]

File: .github/workflows/development-ci.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  push:
+    branches:
+      - development
+  workflow_dispatch:
+    descriptions: "Manually sync Development to Development-CI"
```
Comment (rndquu) [inline .github/workflows/development-ci.yml]: Is it a valid yml key `descriptions`? Can't find any [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) about it

**Review by rndquu — CHANGES_REQUESTED** [review]

**Review by rndquu — APPROVED** [review]

**Review by rndquu — CHANGES_REQUESTED** [review]
check this [comment](https://github.com/ubiquity/ubiquity-dollar/pull/703/files#r1239923867)

File: .github/workflows/development-ci.yml (hunk 2 above)
Comment (BeanieMen) [reply .github/workflows/development-ci.yml]: hmmm why did i do that lol

**Review by rndquu — APPROVED** [review]

**Comment by BeanieMen** [comment]
@zgorizzo69 can you review this pr?

**Comment by rndquu** [comment]
> @zgorizzo69 can you review this pr?

Regarding this PR it is better for @pavlovcik to review

File: .github/workflows/development-ci.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Run merge
+        run: |
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
```
Comment (0x4007) [inline .github/workflows/development-ci.yml:25]: This seems like a hack/work around. Are you sure this is the correct way to do this?

Comment (BeanieMen) [reply .github/workflows/development-ci.yml:25]: yes

**Review by 0x4007 — APPROVED** [review]

**Comment by BeanieMen** [comment]
@pavlovcik sorry, but can i change my receiving wallet address

**Comment by 0x4007** [comment]
The permit was already generated for the previous wallet. You should change now using the wallet command for the next bounty.

**Comment by BeanieMen** [comment]
@pavlovcik i no longer have access to the my previous wallet which i used

**Comment by BeanieMen** [comment]
what should i do

**Comment by 0x4007** [comment]
I need to spend some time looking into permit invalidation again and manually send the payment. Please register your new wallet address now.

**Comment by BeanieMen** [comment]
thank you very much

**Comment by BeanieMen** [comment]
@pavlovcik are you free to address my issue

**Comment by 0x4007** [comment]
@rndquu is working on the permit invalidation. After that's done we can send

**Comment by BeanieMen** [comment]
@rndquu how much time will permit invalidation take

**Comment by rndquu** [comment]
> @rndquu is working on the permit invalidation. After that's done we can send

To invalidate this [permit](https://github.com/ubiquity/ubiquity-dollar/issues/603#issuecomment-1606964861):
1. Open https://gnosisscan.io/address/0x000000000022D473030F116dDEE9F6B43aC78BA3#writeContract and connect your wallet
2. Call `invalidateUnorderedNonces()` with the following params:
```
wordPos: 87440459683411777251988225221611060832059068822132529362254528344230518065
mask: 45671926166590716193865151022383844364247891968
```

**Comment by 0x4007** [comment]
> > @rndquu is working on the permit invalidation. After that's done we can send
> 
> To invalidate this [permit](https://github.com/ubiquity/ubiquity-dollar/issues/603#issuecomment-1606964861):
> 
> 1. Open https://gnosisscan.io/address/0x000000000022D473030F116dDEE9F6B43aC78BA3#writeContract and connect your wallet
> 2. Call `invalidateUnorderedNonces()` with the following params:
> 
> ```
> wordPos: 87440459683411777251988225221611060832059068822132529362254528344230518065
> mask: 45671926166590716193865151022383844364247891968
> ```

https://gnosisscan.io/tx/0x3292dc725f0dddc4ac081f607e1dced4e33a920956f2a629493b49551de9e85c

**Comment by 0x4007** [comment]
https://gnosisscan.io/tx/0xa3d3200177d6654d7205d9e78ad5ed7685eaca683f625ea5bd630ab6a6ba25ae

@me505

**Pull request merged**